data/campaigns/atl02.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,458 @@
+end
+
+
+-- =======================================================================
+--                                Objectives
+-- =======================================================================
+
+obj_find_port = {
+   name = "obj_find_port",
+   title = _ "Find a place to build a port",
+   number = 1,
+   body = objective_text(_"Find a portspace",
+   p(_[[To set foot on this island we first must find a building space where a port can be built. Explore the nearby coast line to find such a building plot.]]) ..
+   p(_[[After the port is build we should cancel the remaining ships expedition, to unload its wares as well.]]))
+}
+
+obj_basic_infrastructure = {
+   name = "obj_basic_infrastructure",
+   title = _ "Ensure the supply of build wares",
+   number = 4,
+   body = objective_text(_"Establish a basic production",
+   p(_[[We will need at least a quarry, a few lumberjacks and a sawmill. A forester might or might not be needed depending on how fast the trees are growing around you. Find the balance.]]))
+}
+
+obj_tools = {
+   name = "obj_tools",
+   title = _ "Build tools quickly",
+   number = 1,
+   body = objective_text(_"Make new tools",
+      p(_[[Most of our tools have been lost in the ship tragedy. Our people will not be able to fulfill their duties without proper tools. Luckily we still have some iron ore and plenty of coal. Build a production of tools, it is a prerequisite for success on island. Don’t forget to build a smelter as well.]]))
+}
+
+obj_mining = {
+   name = "obj_mining",
+   title = _ "Explore the mountains for ressources",
+   number = 3,
+   body = objective_text(_"Build Mines",
+      p(_[[As we are able to make tools again, our supply of mining ressources needs to be replenished soon. Explore the mountains and build at least one mine of each type to mine coal, iron and gold.]]))
+}
+
+obj_explore = {
+   name = "obj_explore",
+   title = _"Explore the surroundings",
+   number = 1,
+   body = objective_text(_"Explore your surroundings",
+   p(_[[We are pretty sure that we are not alone on the island. We do not know if the others on this island are friends or foes so we need to treat carefully. Expand and explore the island by building at least one habitat for a scout and military buildings. Build at least one tower for his bigger view range.]]))
+}
+
+obj_spidercloth = {
+   name = "obj_spidercloth_production",
+   title = _"Build a spider farm and a weaving mill",
+   number = 2,
+   body = objective_text(_"Spidercloth Production",
+      p(_[[We need some spidercloth urgently as we are running out of it. So it is essential to spare every piece we still have. Reduce the input of all buildings that consume spidercloth. Don’t forget the constructionsites.]]) ..
+      p(_[[After that we need to have a farm under all circumstances as it needs spidercloth to be build. Additionally we need a spiderfarm and a weaving mill. Later on a gold spinning mill would be helpful as well.]]))
```

Comment:
```suggestion
      p(_[[We need some spidercloth urgently as we are running out of it. It is therefore essential to spare every piece we still have. Reduce the input of all buildings that consume spidercloth, not forgetting the construction sites.]]) ..
      p(_[[Afterwards, we need to construct a farm under all circumstances, as it needs spidercloth to be built. Additionally, we need a spiderfarm and a weaving mill. Later on, a gold spinning mill would be helpful as well.]]))
```
I'm not sure the gold spinning mill really fits in here as it consumes a totally different type of input